src/sections/footer.html

```diff
@@ -1,3 +1,23 @@
-<div>
-    
-</div>
\ No newline at end of file
+<div class="flex justify-center pb-10 px-9 items-center sticky top-0 pt-8">
+    <img class="md:w-widthImage md:h-heightImage" src="/img/red.webp" alt="The red vector">
```

Comment:
Check the comments I left on the tailwind config then replace these tailwind classes.
Instead of giving the div containing the  image pt gives it a py